control-plane/pkg/reconciler/consumergroup/consumergroup.go

```diff
@@ -41,11 +42,16 @@ import (
 
 	"knative.dev/eventing/pkg/scheduler"
 
+	"knative.dev/eventing-autoscaler-keda/pkg/reconciler/keda"
 	kafkainternals "knative.dev/eventing-kafka-broker/control-plane/pkg/apis/internals/kafka/eventing/v1alpha1"
 	internalv1alpha1 "knative.dev/eventing-kafka-broker/control-plane/pkg/client/internals/kafka/clientset/versioned/typed/eventing/v1alpha1"
 	"knative.dev/eventing-kafka-broker/control-plane/pkg/client/internals/kafka/injection/reconciler/eventing/v1alpha1/consumergroup"
 	kafkainternalslisters "knative.dev/eventing-kafka-broker/control-plane/pkg/client/internals/kafka/listers/eventing/v1alpha1"
 	"knative.dev/eventing-kafka-broker/control-plane/pkg/kafka"
+
+	kedav1alpha1 "knative.dev/eventing-autoscaler-keda/third_party/pkg/apis/keda/v1alpha1"
```

Comment:
Do we really want the `third_party` directory here?

Reply:
A couple of reasons why I used third_party dir here for keda APIs, kedaclientset and also for fake clients in the test files...

1. Within this PR, we're (re)using the existing methods from EAK: https://github.com/knative-sandbox/eventing-autoscaler-keda/blob/main/pkg/reconciler/keda/resources.go#L25 where apis come from third_party and not directly from keda, so that can cause a conflict when passing values. 2. Keda does not [generate](https://github.com/kedacore/keda/blob/main/hack/update-codegen.sh) the injection Knative codegen files but we have it generated in EAK [third_party](https://github.com/knative-sandbox/eventing-autoscaler-keda/tree/main/third_party/pkg/client/injection)

Reply:
Ok, perhaps something for later - if we really want